docs/src/community/goversance.md

```diff
@@ -0,0 +1,34 @@
+# Code of Conduct
+
+Youki follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+# Maintainership
+
+Youki is governed by Maintainers who are elected from active contributors.
+Youki will keep the [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+Maintainers are [here](./maintainer.md).
+
+## Roles
+
+Maintainers are consist of the following roles:
+
+- Committer (Full maintainership)
+    Committers have full write accesses to repos.
+    Committers’ commits should still be made via GitHub pull requests (except for urgent security fixes), and should not be pushed directly.
+
+## Addition and promotion of Maintainers
+
+A contributor who have made significant contributions in quality and in quantity can be also directly invited as a Committer.
+A proposal to add or promote a Maintainer must be approved by 2/3 of the Committers who vote within 7 days.
+Voting needs 2 approvals at least. The proposer can vote too. Votes from the same company will be counted as one vote.
+A proposal should happen as a GitHub pull request to the Maintainer list above. It is highly suggested to reach out to the Committers before submitting a pull request to check the will of the Committers.
```

Comment:
```suggestion
A proposal should be made via a GitHub pull request to the file containing the list of Maintainer defined above. Before submitting the pull request, the proposer should reach out to the Committers to check their willingness.
```

Reply:
...to check their willingness to support the proposal.